src/views/document/RouteView.vue

```diff
@@ -25,7 +26,8 @@
             <div class="column is-4">
               <label-value :label="$gettext('ratings')">
                 <document-rating v-if="$documentUtils.hasRating(document)" :document="document" show-helper />
-                <edit-link v-else :document="document" :lang="$user.lang" />
+                <edit-link v-else-if="isEditable" :document="document" :lang="$user.lang" />
+                <span v-else>...</span>
```

Comment:
Is this the best choice? Looks like a *Idontknowwhattodointhatfuckingcase* 🙂 
What about `undefined` (`non définies`) instead?